packages/template-retail-react-app/package.json

```diff
@@ -85,7 +85,7 @@
         },
         {
             "path": "build/vendor.js",
-            "maxSize": "280 kB"
+            "maxSize": "283 kB"
```

Comment:
This is broken on this branch without making ANY changes. To unblock the release I have to bump the size limit.

I do not want to introduce any code changes to address this issue since this branch is on maintenance mode, and changes may introduce regressions.